src/Components/ServiceScene/LogsJsonScene.tsx

```diff
@@ -599,21 +605,27 @@ export class LogsJsonScene extends SceneObjectBase<LogsJsonSceneState> {
       (field) => field.type === FieldType.other && isLabelTypesField(field.name)
     );
 
+    const templateSrv = getTemplateSrv();
+    const replace = templateSrv.replace.bind(templateSrv);
+
     const timeZone = getTimeZone();
     if (dataFrame && newState.data) {
       const isRerooted = getLineFormatVariable(this).state.filters.length > 0;
+      const derivedFields: Field[] =
+        dataFrame?.fields
+          .filter((f) => f.config.links)
+          .map((field) => ({ ...field, getLinks: getLinksSupplier(dataFrame, field, {}, replace) })) ?? [];
```

Comment:
Are the derivedFields template variables? haven't seen this before getTemplateSrv

Reply:
The table does this as well in `/src/Components/Table/LogLinePill.tsx`, I think it's just in case the links contain template variables